string first_name
string patronym
string last_name
---
string full_name
